amq-message-age/README.md.jinja

```diff
@@ -34,9 +39,13 @@ These dashboards accompany [SignalFx's ActiveMQ integration](https://github.com/
 | Java | 1.5 or later |
 | Maven | (match with Java version) |
 
+## SETUP
+
 ### INSTALLATION
 
-1. Download SignalFx's ActiveMQ message age listener from <a target="_blank" href="https://github.com/signalfx/activemq-integration">https://github.com/signalfx/activemq-integration</a>.
+To access this tool, perform the following steps:
+
+1. Download the SignalFx ActiveMQ message age listener from <a target="_blank" href="https://github.com/signalfx/activemq-integration">https://github.com/signalfx/activemq-integration</a>.
```

Comment:
See previous comment about the name of the tool.

Reply:
revised